src/popups/GeltungsbereichePopup/GeltungsbereichePopup.js

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import Feature from 'ol/Feature';
+import { Layer } from 'mobility-toolbox-js/ol';
+import { Typography, makeStyles } from '@material-ui/core';
+import geltungsbereicheMapping from '../../utils/geltungsbereicheMapping.json';
+
+const useStyles = makeStyles(() => {
+  return {
+    subtext: {
+      color: '#888',
+    },
+  };
+});
+
+const propTypes = {
+  feature: PropTypes.instanceOf(Feature).isRequired,
+  layer: PropTypes.instanceOf(Layer).isRequired,
+};
+
+const defaultProps = {};
+
+const GeltungsbereichePopup = ({ feature, layer }) => {
+  const classes = useStyles();
+  const geltungsbereiche = JSON.parse(feature.get('geltungsbereiche'));
+  const topic = useSelector((state) => state.app.activeTopic);
+  const layers = topic.layers.filter((l) => {
+    return /^ch.sbb.geltungsbereiche-/.test(l.key);
+  });
+
+  useEffect(() => {
+    // Shift select style tu current feature
+    layers.forEach((l) => l.select([]));
+    if (layer) {
+      layer.select([feature]);
+    }
+  }, [layers, layer, feature]);
+
+  return (
+    <div className="wkp-geltungsbereiche-popup">
+      {geltungsbereiche &&
+        Object.keys(geltungsbereiche).map((product) => {
+          return (
+            <React.Fragment key={product}>
+              <Typography paragraph>
+                <b>{geltungsbereicheMapping[product]}</b>
+                <br />
+                <span className={classes.subtext}>
+                  {geltungsbereiche[product].reduce(
+                    (providers, provider, idx, arr) =>
+                      `${providers}${`${provider}${
+                        idx !== arr.length - 1 ? ', ' : ''
+                      }`}`,
+                    '',
+                  )}
+                </span>
+              </Typography>
+            </React.Fragment>
+          );
+        })}
+    </div>
+  );
+};
+
+GeltungsbereichePopup.propTypes = propTypes;
+GeltungsbereichePopup.defaultProps = defaultProps;
+
+GeltungsbereichePopup.renderTitle = (f, t) => {
+  return t('ch.sbb.geltungsbereiche');
```

Comment:
Do we really want the same name on each popup ?

Reply:
I meant, using also the layer's name, because right now we don't know if the selected feature si a bus or a train.
Example:
```
Geltungsbereiche BETA:  bus
```